configs/config.js

```diff
@@ -4,27 +4,34 @@ import {
   homepage,
   license,
   name,
-  version
+  version,
 } from '../package.json'
 
+const loose = true
+
 const babelSetup = {
   babelrc: false,
-  presets: [['es2015', { modules: false }]],
-  plugins: ["transform-class-properties"],
-  exclude: 'node_modules/**'
+  presets: [['@babel/preset-env', { modules: false, loose }]],
+  plugins: [['@babel/plugin-proposal-class-properties', { loose }]],
+  exclude: 'node_modules/**',
 }
 
 const uglifyOutput = {
+  compress: {
+    pure_getters: true,
+    unsafe: true,
+  },
+  toplevel: true,
```

Comment:
after a thought - this actually should be done only for `es` build

or you could create separate umd and cjs, where cjs could benefit from that too, but this is not safe for umd, because some vars are hoisted outside of the umd closure, see this:
```js
var e,t;e=this,t=function()
```
within browser env (if you just use the script without bundling it with a tool like webpack or something) this can override e & t global variables

Reply:
OTOH im not sure if uglifying cjs/es is really a good practice, checking the source code in your node_modules becomes really cumbersome and consumers' that use bundling should have their minifier set up anyway

uglifying for ready to be used as script tag umd bundle is still OK ofc

Reply:
@Andarist What great insights. I will make updates based on your comments. ~THANK YOU!